src/artm/core/master_component.h

```diff
@@ -94,8 +95,8 @@ class MasterComponent : boost::noncopyable {
 
   void CreateOrReconfigureRegularizer(const RegularizerConfig& config);
 
+  void AddDictionary(std::shared_ptr<Dictionary> dictionary);
```

Comment:
Move `AddDictionary` to `private:` section? It seems to be implementation detail of master component.